# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nsxiv/hotfix.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nsxiv/config.mk.vanilla	2025-01-09 16:41:41.527457510 -0500
+++ nsxiv/config.mk	2025-01-09 16:44:26.017841918 -0500
@@ -17,10 +17,11 @@
 HAVE_LIBEXIF  = $(OPT_DEP_DEFAULT)
 
 # CFLAGS, any additional compiler flags goes here
-CFLAGS = -Wall -pedantic -O2 -DNDEBUG
+CFLAGS = -Wall -pedantic -O2 -DNDEBUG $(shell pkg-config --cflags imlib2)
 # Uncomment for a debug build using gcc/clang
 # CFLAGS = -Wall -pedantic -DDEBUG -g3 -fsanitize=address,undefined
 # LDFLAGS = $(CFLAGS)
+LDFLAGS=$(shell pkg-config --libs imlib2 x11)
 
 # icons that will be installed via `make icon`
 ICONS = 16x16.png 32x32.png 48x48.png 64x64.png 128x128.png
